app/views/play/level/PlayLevelView.coffee

```diff
@@ -147,6 +148,14 @@ module.exports = class PlayLevelView extends RootView
       @load()
       application.tracker?.trackEvent 'Started Level Load', category: 'Play Level', level: @levelID, label: @levelID unless @observing
 
+    @calcTimeOffset()
+    @mandate = @supermodel.loadModel(new Mandate()).model
+
+    if features.china
+      setTimeout =>
+        @checkTournamentEndInterval = setInterval @checkTournamentEnd(), 10000
```

Comment:
Same comment as above. This won't execute. Use arrow function to make this lazy or bind it.

Reply:
Ooops. It returns itself as a definition. I'm not sure this pattern is ideal.